Content.Server/Construction/PartExchangerSystem.cs

```diff
@@ -83,9 +140,6 @@ private void OnDoAfter(EntityUid uid, PartExchangerComponent component, DoAfterE
             storage.Storage.Insert(unused.Owner);
             _storage.Insert(uid, unused.Owner, null, false);
         }
-        _construction.RefreshParts(args.Args.Target.Value, machine);
-
-        args.Handled = true;
```

Comment:
Why were these removed?

Reply:
Machine frames don't have a RefreshParts function and args are handled at the new line 52 in the OnDoAfter